### ISSUE TITLE
Not compatible with .NET 8?

### ISSUE THREAD

**Issue by sliekens** [johnkors/MinimalHttpLogger#7]
Hi, I am experimenting with this package, but I don't see any minimal logging in .NET 8?

I'm pretty sure my Program looks good:

``` cs
var builder = Host.CreateApplicationBuilder(args);
builder.AddServiceDefaults(); // Aspire

builder.Services.AddHttpClient<ISomeClient, SomeClient>()
    .ConfigurePrimaryHttpMessageHandler(() => new SocketsHttpHandler
    {
        AllowAutoRedirect = true,
        AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate,
        MaxConnectionsPerServer = 20,
        ConnectTimeout = TimeSpan.FromSeconds(10),
        PooledConnectionLifetime = TimeSpan.FromMinutes(5),
        MaxAutomaticRedirections = 7
    });

// Add the sauce
builder.Services.UseMinimalHttpLogger();

var host = builder.Build();

await host.RunAsync();
```

My appsettings:

``` json
{
  "Logging": {
    "LogLevel": {
      "Default": "Information",
      "Microsoft.Hosting.Lifetime": "Information",
      "System.Net.Http.HttpClient": "Warning",
      "Polly": "Warning"
    }
  }
}
```  

I dunno what I'm missing. I don't see the minimal logging. If I change the `System.Net.Http.HttpClient` logging back to `Information`, it still shows me all of the verbose Microsoft logging. This is a clue to me that the logger is not being replaced.

Any hints?

**Comment by johnkors** [comment]
Tried adding your custom HttpClient with the sample named `With`:

![snip Ua2ugZEq@2x](https://github.com/user-attachments/assets/86f157ea-209a-4f73-bd9a-d5e9152a42ce)

Seems to work fine..? Try it yourself by modifying the samples, perhaps?

**Comment by sliekens** [comment]
Thanks, I tried the sample, it worked, so I tried my own code again, and the problem mysteriously disappeared. 😵

Part of the reason why it didn't (appear to) work was my logging config:

```
      "System.Net.Http.HttpClient": "Warning",
```

I wrongly assumed the minimal logger name would start with `"MinimalHttpLogger"` but this config also hides all of the minimal logging.

Closing as not a real issue, thanks.
